libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/JSONUtils.java

```diff
@@ -213,6 +213,11 @@ public static boolean getBool(JSONObject json, String name) {
         return true;
     }
 
+    public static boolean isStringTrue(JSONObject json, String name) {
```

Comment:
This appears very similar to the `getBool()` method above it. Can they be combined?